packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1608,11 +1645,17 @@ class StackViewport extends Viewport implements IStackViewport {
 
     // set voi for the first time
     const { windowCenter, windowWidth } = imagePixelModule;
-    const voiRange =
+    let voiRange =
```

Comment:
So what happens is this ends up `undefined`? Shouldn't we then default to the range of the image? (min and max pixel values)?

Reply:
if undefined, it will get the range of pixels in the setVOIRange